client/SDL/CMakeLists.txt

```diff
@@ -98,10 +98,29 @@ set(LIBS
         freerdp-client
     )
 
+option(WITH_WEBVIEW "Build with QtWebEngine support for AAD login popup browser" OFF)
+if (WITH_WEBVIEW)
+	include(FindPkgConfig)
+	pkg_check_modules(QT Qt5WebEngineWidgets)
```

Comment:
might be a good idea to just add `REQUIRED` here and remove the found check below. If we already request `WITH_WEBVIEW=ON` manually then expect it is available and if not fail.